app/views/middleware_topology/show.html.haml

```diff
@@ -10,7 +10,7 @@
         %svg.kube-topology
           %g.MiddlewareServer{:transform => "translate(21, 21)"}
             %circle{:r => "17"}
-            %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => "/assets/100/middleware_server.png", :y => "-9"}
+            %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => "/assets/svg/vendor-wildfly.svg", :y => "-9"}
```

Comment:
My problem is not the difference between generic and specific, but that path as it is should never work from a browser, unless you disable hashes on asset names. You really need to wrap that value in `image_path` to get paths that will work on the server.